es6/es6.tag

```diff
@@ -0,0 +1,10 @@
+<es-6>
+
+  <p>{ message }</p>
+
+  <script type="es6">
+    const name = 'Riot'
+    this.message = `Hello ${name}`
```

Comment:
Right. But Riot has own `es6`-like syntax. It could make the users confused. Thoughts? (I saw a few issue about this)